### ISSUE TITLE
More than one WhatsApp

### ISSUE THREAD

**Issue by eaba** [IwraStudios/Whatsapp-Bot#1]
Can this handle more than one WhatsApp numbers?

**Comment by IwraStudios** [comment]
Do you mean as host or as clients? please explain

**Comment by eaba** [comment]
As clients.
Creating a service where users can automate their whatsapp communications.
I basically want to know if it is possible to create multiple unique instances of your api, without whatsapp seeing those QR as coming from the same browser. Is it possible to make the browser unique to each instances........I hope you get the idea though i find it hard explaining my goal

**Comment by IwraStudios** [comment]
You can create multiple instances, by instantiating a new FirefoxWApp() and then Starting the driver (basically running two of the FirefoxExample's at the same time. This does mean that automatic cookie saving won't work(it isn't implemented completely yet), so you will have to login every time you start the driver, we may improve on this in the future.

TL;DR
you can, but future features may be limited like: "session/cookie auto-saving".

**Comment by eaba** [comment]
Can I save(myself) these cookies and use them every time I create new instances. I am thinking of thousands of instances. 
Does these cookies have expiration?

**Comment by IwraStudios** [comment]
You can but I'll have to implement "manual cookie filenames", this won't take long but I'm really busy. I'll see if I can have it done by tomorrow.

**Comment by eaba** [comment]
That will be appreciated!

> Do you mean as host or as clients? please explain

the host part of it, how would that work?

**Comment by IwraStudios** [comment]
I meant if you meant that multiple people would need to communicate with it(client), or that you need to have multiple browsers/drivers open at the same time(host)

so you saying:
> I basically want to know if it is possible to create multiple unique instances of your api

Makes it a host side problem

I know for a fact that the API can handle a lot of numbers(clients) but I don't know how many

**Comment by eaba** [comment]
Okay.
I appreciate your time and efforts!

**Comment by IwraStudios** [comment]
I've implemented the Save/Load functions which take a parameter of "string" which is just the path to the save-file

I've not tested it yet so if you have any problems please notify me

**Comment by eaba** [comment]
Thanks
